docs/elements.md

```diff
@@ -1268,13 +1268,13 @@ The overlay element allows existing PrairieLearn and HTML elements to be layered
 
 ```html
 <pl-overlay width="400" height="400" clip="false">
-    <pl-location left="0" top="0" halign="left" valign="top">
+    <pl-background>
```

Comment:
Missing `pl-background` in documentation even though it's a specific version of `pl-location`, still useful to have its own table.

Reply:
Yep, though the `pl-background` child doesn’t have any attributes that can get set.

Reply:
Eh, maybe an empty table with a line underneath saying no attributes are necessary? It's handled by `pl-drawing`? I'm just worried that folks will overlook the brief mention in the details section.